apps/server/src/modules/board/repo/board-node.builder-impl.ts

```diff
@@ -84,6 +86,19 @@ export class BoardNodeBuilderImpl implements BoardNodeBuilder {
 		this.registerNode(textElementNode);
 	}
 
+	buildFileElementNode(fileElement: FileElement, parentId?: string | undefined, position?: number | undefined): void {
```

Comment:
In which case can the parent be undefined?

Reply:
This is for enabling root nodes. We currently do not allow elements as root nodes, but this is a decision of the business logic (domain layer). We still have to be able to save any object as a root node (if we we need to).